start.sh

```diff
@@ -88,6 +88,8 @@ chat_model_stem=$(basename "$chat_model_name" .gguf)
 chat_ctx_size=$(awk -F'"' '/"chat_ctx_size":/ {print $4}' config.json)
 # parse prompt type for chat model
 prompt_type=$(awk -F'"' '/"prompt_template":/ {print $4}' config.json)
+# parse system prompt for chat model
+system_prompt=$(awk -F'"' '/"rag_prompt":/ {print $4}' config.json)
```

Comment:
This should be `rag_prompt`

Reply:
Yes. It satisfies this requirment: If scored points are present, namely RAG context returned, then use `rag_prompt` to replace the system prompt given by users.

If the requirement is not inappropiate, please correct me. Thanks!

Reply:
Yes. The behavior is correct. But I think the argument name should be called `--rag-prompt` in the rag-api-server.wasm CLI.

Reply:
Accepted. Will rename the CLI argument to `--rag-prompt` in `rag-api-server`, but the change will be in the next release of `rag-api-server`.